### ISSUE TITLE
jampack breaks height of inlined svg

### ISSUE THREAD

**Issue by ldub** [divriots/jampack#162]
when you run jampack and you have something like

```
          <img height="50px"
            src="/img/my-logo.svg"
            alt="my logo"
          />
```

jampack will output something like

```
<img height="50px" src="data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' color-inter... omitted" alt="my logo" fetchpriority="high">
```

I guess the height attribute does not work for inline SVGs and you end up with the wrong size svg. to fix it you must assign a class and set the height via CSS.

This isn't a huge deal - but I imagine you want jampack to never break the layout, and this does break it, so I thought I'd report.

**Comment by ldub** [comment]
seems like https://github.com/divriots/jampack/issues/144 might be related

**Comment by georges-gomes** [comment]
> I imagine you want jampack to never break the layout, and this does break it, so I thought I'd report.

I agree so I will add a test case and look into it.
Thanks!

---

Side note: I don't know if `height="50px"` is valid HTML. I think only `height="50"` is.

**Comment by georges-gomes** [comment]
This should be fixed in `0.30.0` - If the `<img>` attribute has `width` or `height`, jampack won't embed. 

It would be nice to embed in a much long PR later.
